src/utils/request.js

```diff
@@ -0,0 +1,16 @@
+// counter used for identifying unique request and it will share among all transports
+let requestCounter = 0;
+
+function resetCounter() {
+    requestCounter = 0;
+}
+
+function getRequestId() {
+    return ++requestCounter;

```

Comment:
@lukeapage I'm using same logic for request counter increment as before, where first request id is 1. Do you remember if it was requirement or can we move to starting request ids from 0?

It would make some unit test cleaner, that's all.

Reply:
I don't recall it being a requirement or why it should be from 1.

Reply:
Ok, will confirm with openapi, and 0 is valid x-request-id, I'll change the implementation to use it.